composer.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "ezsystems/ezplatform-i18n",
+  "description": "eZ Platform translations",
```

Comment:
Maybe elaborate a bit, to make it clear that this is NOT what you would install to actually get translations.

Or is it ? If you want a lot of languages, would you install this one instead of the individual language packages ?